### PR TITLE
scheduler: handle err and remove useless assignment

### PR DESCRIPTION
**Pull request by shinytang6**
**What type of PR is this?**
/sig scheduling
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -537,12 +537,13 @@ func (sched *Scheduler) scheduleOne() {
 		for _, pl := range plugins.PrebindPlugins() {
 			approved, err := pl.Prebind(plugins, assumedPod, scheduleResult.SuggestedHost)
 			if err != nil {
-				approved = false
```
Comment (yastij) [inline pkg/scheduler/scheduler.go]: what happens if a plugin returns `true, err` (which shouldn't happen), this is more of defensive check I'd leave it here.

**Review by yastij — CHANGES_REQUESTED** [review]

File: pkg/scheduler/scheduler.go (hunk 1 above)
Comment (shinytang6) [reply pkg/scheduler/scheduler.go]: Okay. l will turn it back

**Comment by shinytang6** [comment]
@denkensk @yastij  PTAL :)

**Review by bsalamat — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**